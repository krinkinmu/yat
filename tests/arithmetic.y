print 10 * 20;
print 10 / 20;
print 20 / 10;
print 20 % 10;
print 5 % 3;
print 10 - 20;
print 20 - 10;
print 20 + 10;
print -10;
print -20;
if (!10) {
    print -10;
} else {
    print 10;
}
if (!20) {
    print -20;
} else {
    print 20;
}
