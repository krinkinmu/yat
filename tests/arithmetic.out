200
0
2
0
2
-10
10
30
-10
-20
10
20
